extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueLightHandler.java

```diff
@@ -273,6 +273,8 @@ public void handleCommand(ChannelUID channelUID, Command command) {
                 break;
         }
         if (lightState != null) {
+            lastSentBrightness = lightState.getBrightness();
```

Comment:
I doubt that this is the correct fix - usually, lightState will only contain brightness OR colorTemperature information. Therefore I assume that you are actually deleting at least one cached information every time, which is not desired.

Reply:
Good point. I was not aware that either `colorTemperature` **OR** `brightness` where sent. I have changed my variables to `Integer` and only cache them if they have been set.